zaza/openstack/charm_tests/octavia/tests.py

```diff
@@ -453,3 +453,36 @@ def test_create_loadbalancer(self):
 
         # If we get here, it means the tests passed
         self.run_resource_cleanup = True
+
+class VolumeAmphoraTest(LBAASv2Test):
+    """LBaaSv2 Volume-based Amphora tests"""
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for LBaaSv2 Volume-based amphora tests
+           only for Openstack Train release and newer releases.
+         """
+        if (openstack_utils.get_os_release() <
+                openstack_utils.get_os_release('bionic_train')):
+            return
```

Comment:
It would probably be better to use `unittest.skip("reason here")` to indicate that the class is being skipped.

Reply:
Thank you for reviewing this and for pointing out the corrections I needed to make.